src/main/java/com/Elessar/app/util/kafka/KafkaConstants.java

```diff
@@ -0,0 +1,14 @@
+package com.Elessar.app.util.kafka;
+
+/**
+ * Created by Hans on 3/17/19.
+ */
+public interface KafkaConstants {
```

Comment:
For now I would recommend just hard code those values when create consumer and producers as in real life we will pass in a config file specifying those values, which allows customization for different client/server if necessary.